src/routes/Place.js

```diff
@@ -66,6 +78,15 @@ export const Place = () => {
   const topPhoto = googleInfo.photos?.length > 0 ? googleInfo.photos[0] : null;
   document.title = 'Hudson Happs | ' + name
   return (
+    <>
```

Comment:
<img width="375" alt="Screen Shot 2023-07-19 at 9 31 54 PM" src="https://github.com/alexsaltstein/HobokenEventsWebApp/assets/25209141/8a0f8d6c-97e0-438c-bca8-ab5b567d5e7a">


What do we want button color to be @alexsaltstein ?